crossbeam-channel/src/channel.rs

```diff
@@ -274,6 +274,13 @@ pub fn never<T>() -> Receiver<T> {
     }
 }
 
+/// Creates a receiver that is disconnected always.
```

Comment:
I'm happy to expand this doccomment with an example, which is in the pr description with polishes, once it's confirmed that this pr has some hope to be merged.